posters.json

```diff
@@ -16495,7 +16495,7 @@
    "presenter": "Alexandre Gramfort",
    "categories": "Neuroinformatics and Data Sharing<br>Databasing and Data Sharing",
    "videochat": "<a href=\"https://meet.jit.si/ohbm2020-1940\" target=\"_ohbm2020-1940\">jitsi:ohbm2020-1940</a>",
-   "pdf": ""
+   "pdf": "https://doi.org/10.5281/zenodo.3891834"
```

Comment:
This should be in posters-overrides.json, not posters.json, which is automatically generated.

Reply:
yeap... I was worrying in making it "readable" ;)  @sappelhoff please redo, sorry for inconveniences

Reply:
see #18 and thanks for doing this @effigies and @yarikoptic!